app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def index
+    render html: '<div>Here is the list of all his posts</div>'.html_safe
+  end
```

Comment:
- [ ] Let's add corresponding view files for each action in your controllers instead of hard coding these render statements.  .
You should create following files:

    `# app/views/users/index.html.erb`
    `# app/views/users/show.html.erb`

    `# app/views/posts/index.html.erb`
    `# app/views/posts/show.html.erb`

Reply:
Thank you mate.